tests/common_for_tests.py

```diff
@@ -82,6 +82,7 @@ def sensor_bands_info():
 def multi_raster_16b():
     source_raster = make_test_raster(4200, ['green', 'red', 'nir', 'blue'], dtype=np.uint16)
     array = source_raster.image.data
+    array.setflags(write=1)
```

Comment:
Perhaps for the sake of the tests, we could do `array.setflags(write=1)` inside `make_test_raster`, just before returning?